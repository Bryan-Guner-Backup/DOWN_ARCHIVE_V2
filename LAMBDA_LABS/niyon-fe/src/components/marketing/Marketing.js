import React from 'react'
import Navbar from '../navbar/Navbar'
import MarketingButtons from './MarketingButtons'

function Marketing () {
  return (
    <div data-test="marketing">
      <Navbar />
      <MarketingButtons />
    </div>
  )
};

export default Marketing
