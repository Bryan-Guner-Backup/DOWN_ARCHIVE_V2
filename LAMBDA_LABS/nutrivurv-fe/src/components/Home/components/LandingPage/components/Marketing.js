import React from "react";
import { Link } from "react-router-dom";
import { ReactComponent as LandingImage } from "../../../../../assets/Splash.svg";

const Marketing = () => {
  return (
    <div className="marketing" style={{ marginBottom: "20vh" }}>
      <section className="pl-lg-5">
        <div className="py-5 pl-5 ml-xl-5 mb-2">
          <h1 className="font-weight-bold mt-3 text-responsive mb-3 ml-xl-5 pl-xl-5">
            Tracking your calories
            <br />
            just got easier.
          </h1>
          <p className="second-text-responsive mb-3 ml-xl-5 pl-xl-5">
            Nutrivurv is an easy to use motivating <br />
            food and nutrition tracking app.
          </p>
          <Link type="button" to="/onboarding" className="pl-xl-5">
            <button
              id="onboarding-button"
              className="btn btn-primary ml-xl-5"
              style={{ width: "15rem" }}
            >
              Sign Up
            </button>
          </Link>
        </div>
        <figure className="mt-1 ml-5 pb-5">
          <LandingImage className="lp-img pl-5 pl-lg-0 ml-xl-n5" />
        </figure>
      </section>
    </div>
  );
};

export default Marketing;
