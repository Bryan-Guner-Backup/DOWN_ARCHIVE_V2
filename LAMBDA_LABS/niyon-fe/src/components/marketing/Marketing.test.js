import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import Marketing from './Marketing'
import findByTestAttr from '../../tests/utils'
// import Navbar from '../navbar/Navbar';
// import Marketingbuttons from './Marketingbuttons';

describe('<Marketing /> component testing', () => {
  let component
  beforeEach(() => {
    component = shallow(<Marketing />)
  })
  it('should match the snapshot testing', () => {
    expect(toJSON(component)).toMatchSnapshot()
  })

  it('should render the <Marketing /> component correctly', () => {
    expect(component.exists()).toBe(true)
    const marketingDiv = findByTestAttr(component, 'marketing')
    expect(marketingDiv.length).toBe(1)
  })

  it('should render <Navbar /> component correctly', () => {
    expect(component.find('Navbar').exists()).toBe(true)
  })

  it('should render <MarketingButtons /> component correctly', () => {
    expect(component.find('MarketingButtons').exists()).toBe(true)
  })
})
