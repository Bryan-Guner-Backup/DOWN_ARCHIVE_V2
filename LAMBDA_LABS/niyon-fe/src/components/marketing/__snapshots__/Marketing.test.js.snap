// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Marketing /> component testing should match the snapshot testing 1`] = `
<div
  data-test="marketing"
>
  <Navbar />
  <MarketingButtons />
</div>
`;
